JournalMonitor/JournalMonitor.cs

```diff
@@ -3138,16 +3138,25 @@ async void ModuleArrived()
                                     {
                                         List<string> items = new List<string>();
                                         List<Module> modules = new List<Module>();
-                                        if (itemsVal is List<object> moduleEdNames)
+                                        if (itemsVal is List<object> itemEDNames)
                                         {
-                                            foreach (string moduleEdName in moduleEdNames)
+                                            foreach (string itemEDName in itemEDNames)
                                             {
-                                                items.Add(moduleEdName);
-                                                var module = Module.FromEDName(moduleEdName);
-                                                if (module != null)
+                                                var item = itemEDName;
```

Comment:
Nit: can we use `itemEDName` like in the `else if` below?

Reply:
So delete line 3145 and change line 3146 from "item" to "itemEDName"?

Reply:
Yes :)

Reply:
Done.